core/app/models/spree/store_credit.rb

```diff
@@ -42,6 +42,8 @@ class StoreCredit < Spree::Base
     extend Spree::DisplayMoney
     money_methods :amount, :amount_used
 
+    self.whitelisted_ransackable_attributes = %w[user_id created_by_id amount currency type_id store_id]
```

Comment:
```suggestion
    self.whitelisted_ransackable_attributes = %w[user_id created_by_id amount currency type_id]
```

@aleksandarpetrushev let's also add associations - type, user, created_by

Reply:
`store_id` filtering will not work as it's already scoped by current store in API and Admin